server/data/placementRequestClient.ts

```diff
@@ -23,10 +24,11 @@ export default class PlacementRequestClient {
     >
   }
 
-  async dashboard(): Promise<Array<PlacementRequest>> {
-    return (await this.restClient.get({ path: paths.placementRequests.dashboard.pattern })) as Promise<
-      Array<PlacementRequest>
-    >
+  async dashboard(isParole: boolean): Promise<Array<PlacementRequest>> {
+    return (await this.restClient.get({
+      path: paths.placementRequests.dashboard.pattern,
+      query: createQueryString({ isParole }),
```

Comment:
I was under the impression (could be wrong!) that you can just do `query: {isParole},` but this is valid too